components/tooltip/index.css

```diff
@@ -38,50 +35,24 @@ governing permissions and limitations under the License.
 	--spectrum-tooltip-spacing-block-start: var(--spectrum-component-top-to-text-75);
 	--spectrum-tooltip-spacing-block-end: var(--spectrum-component-bottom-to-text-75);
 
-	/* icon spacing */
-	--spectrum-tooltip-icon-spacing-inline-start: var(--spectrum-text-to-visual-75);
-	--spectrum-tooltip-icon-spacing-inline-end: var(--spectrum-text-to-visual-75);
-	--spectrum-tooltip-icon-spacing-block-start: var(--spectrum-component-top-to-workflow-icon-75);
-
-	/* colors */
-	--spectrum-tooltip-background-color-informative: var(--spectrum-informative-background-color-default);
-	--spectrum-tooltip-background-color-positive: var(--spectrum-positive-background-color-default);
-	--spectrum-tooltip-background-color-negative: var(--spectrum-negative-background-color-default);
-
-	--spectrum-tooltip-content-color: var(--spectrum-white);
+	--spectrum-tooltip-content-color: var(--spectrum-gray-25);
+	--spectrum-tooltip-background-color-default: var(--spectrum-neutral-background-color-default);
 
 	/* tip */
-	--spectrum-tooltip-tip-inline-size: var(--spectrum-tooltip-tip-width);
 	--spectrum-tooltip-tip-block-size: var(--spectrum-tooltip-tip-height);
 
 	/* Width and height of square element used to render the tip triangle. */
-	--spectrum-tooltip-tip-square-size: var(--spectrum-tooltip-tip-inline-size);
+	/* To get a square that measure 10px on the diagonal, the sides have to be 7px. */
+	--spectrum-tooltip-tip-square-size: 7px;
```

Comment:
Yay math and Pythagorean theorem! I wonder if it's helpful to explain further where that number comes from, but I think that's really up to you, in my opinion this comment does make it possible to figure out why it's 7.

For the large platform scale, I think this ends up being `1px` off (because `tooltip-tip-height` is `6px` rather than `5px`, and the square root of 6 squared + 6 squared is about 8.48... I'm not sure this is significant enough to address though, and visually there aren't any issues being caused by that 1px difference. It's nice that that's the only size change that we'd need to think about!

Reply:
Great callout on the large platform scale. I'll add a custom token for this.